### ISSUE TITLE
Cannot read `TTree` from file

### ISSUE THREAD

**Issue by gipert** [tamasgal/UnROOT.jl#116]
Hi! First of all thank you for providing the package, hope to see it grow and improve in the future!

I just started trying it out and I've found (I think) a bug. I cannot read the `TTree`stored in the attached file [test.root.txt](https://github.com/tamasgal/UnROOT.jl/files/7236227/test.root.txt) (I had to add `.txt` otherwise GitHub does not let me upload it). In particular:
```julia
julia> using UnROOT

julia> f = ROOTFile("test.root")
ROOTFile with 3 entries and 24 streamers.
mage-output.root
├─ InputMacroCommands (TObjString)
└─ NumberOfPrimaries (TParameter<long>)


julia> keys(f)
3-element Vector{String}:
 "InputMacroCommands"
 "NumberOfPrimaries"
 "fTree"

julia> t = LazyTree(f, "fTree")
ERROR: MethodError: no method matching keys(::Nothing)
Closest candidates are:
  keys(::Union{Tables.AbstractColumns, Tables.AbstractRow}) at /home/gipert/.julia/packages/Tables/i6z2B/src/Tables.jl:184
  keys(::Dictionaries.AbstractIndices) at /home/gipert/.julia/packages/Dictionaries/xfzRj/src/AbstractIndices.jl:38
  keys(::OrderedCollections.OrderedSet) at /home/gipert/.julia/packages/OrderedCollections/PRayh/src/ordered_set.jl:95
  ...
Stacktrace:
 [1] LazyTree(f::ROOTFile, s::String)
   @ UnROOT ~/.julia/packages/UnROOT/EoYKp/src/iteration.jl:258
 [2] top-level scope
   @ REPL[18]:1
```
Any idea?

**Comment by aminnj** [comment]
Thanks for the file! Indeed there's something funny going on. For now, you can "solve" it with a bandaid:
```diff
diff --git a/src/streamers.jl b/src/streamers.jl
index 4b14ffb..6ab8144 100644
--- a/src/streamers.jl
+++ b/src/streamers.jl
@@ -589,6 +589,9 @@ function unpack(io, tkey::TKey, refs::Dict{Int32, Any}, ::Type{T}) where {T<:ROO
     mod, typename = split(String(streamer_name), ".")
     streamer = getfield(@__MODULE__, Symbol(typename))
     readfields!(cursor, fields, streamer)
+    if streamer === TBranchElement_10
+        fields[:fIOFeatures] = 0x00
+    end
     streamer(;cursor=cursor, fields...)
 end
```
since the stacktrace (after removing some `try-catch` which hides it from the user) is 
```julia
julia> f["fTree"]
fields[:fIOFeatures] = missing
ERROR: Field 'fIOFeatures' has no default, supply it with keyword.
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:33
 [2] unpack(io::IOBuffer, tkey::UnROOT.TKey32, refs::Dict{Int32, Any}, #unused#::Type{UnROOT.TBranchElement})
   @ UnROOT ~/.julia/dev/UnROOT/src/streamers.jl:592
 [3] readobjany!(io::IOBuffer, tkey::UnROOT.TKey32, refs::Dict{Int32, Any})
   @ UnROOT ~/.julia/dev/UnROOT/src/streamers.jl:239
 [4] unpack(io::IOBuffer, tkey::UnROOT.TKey32, refs::Dict{Int32, Any}, T::Type{UnROOT.TObjArray})
   @ UnROOT ~/.julia/dev/UnROOT/src/streamers.jl:316
 [5] UnROOT.TTree(io::IOStream, tkey::UnROOT.TKey32, refs::Dict{Int32, Any}; top::Bool)
   @ UnROOT ~/.julia/dev/UnROOT/src/bootstrap.jl:915
 [6] UnROOT.TTree(io::IOStream, tkey::UnROOT.TKey32, refs::Dict{Int32, Any})
   @ UnROOT ~/.julia/dev/UnROOT/src/bootstrap.jl:857
 [7] _getindex(f::ROOTFile, s::String)
   @ UnROOT ~/.julia/dev/UnROOT/src/root.jl:151
 [8] getindex(f::ROOTFile, s::String)
   @ UnROOT ~/.julia/dev/UnROOT/src/root.jl:128
 [9] top-level scope
   @ REPL[7]:1
```
will take a deeper look in a few more hours. Also, maybe others know the problem?

**Comment by aminnj** [comment]
Solved in master/v0.6.3!

**Comment by gipert** [comment]
Super fast! Thanks a lot.
